incubator/hnc/Makefile

```diff
@@ -50,6 +50,9 @@ fmt:
 vet:
 	go vet ./...
 
+kustomize:
```

Comment:
Maybe test-kustomize? It doesn't actually do anything useful other than testing.